src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,79 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { memo, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAddMovie: (movies: Movie[]) => void,
+  visibleMovies: Movie[];
+};
+
+export const FindMovie: React.FC<Props> = memo(({
+  onAddMovie,
+  visibleMovies,
+}) => {
+  const [query, setQuery] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const getMovieFromServer = async () => {
+    setLoading(true);
+
+    const movieFromServer = await getMovie(query);
+
+    try {
+      if ('Title' in movieFromServer) {
+        setLoading(true);
+
+        setMovie({
+          title: movieFromServer.Title,
+          description: movieFromServer.Plot,
+          imgUrl: movieFromServer.Poster === 'N/A'
+            ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+            : movieFromServer.Poster,
+          imdbUrl: `https://www.imdb.com/title/${movieFromServer.imdbID}`,
+          imdbId: movieFromServer.imdbID,
+        });
+      } else {
+        setIsError(true);
+        setLoading(false);
```

Comment:
You don't need to add this `setLoading(false)`, because you have it in `finally`, which will work anyway, after your promise will end